control_plane/src/bin/neon_local.rs

```diff
@@ -798,6 +798,16 @@ fn handle_endpoint(ep_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<(
                 ep.start(&auth_token, safekeepers, remote_ext_config)?;
             }
         }
+        "sighup" => {
```

Comment:
you also need to make changes in `compute_ctl` when new config arrives

Reply:
`compute_ctl` already has reconfigure which applies config